labs/module4/English/C/README.ipynb

```diff
@@ -300,7 +300,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pgcc -fast -ta=tesla -Minfo=accel -o laplace_data_clauses jacobi.c laplace2d.c && ./laplace_data_clauses 1024 1024"
+    "!pgcc -fast -ta=tesla,managed -Minfo=accel -o laplace_data_clauses jacobi.c laplace2d.c && ./laplace_data_clauses 1024 1024"
```

Comment:
I believe at this point in the tutorial we want the managed flag removed to necessitate the addition of data clauses.

Reply:
removed managed flag. I sent the wrong build.